client/src/components/overview/overview.jsx

```diff
@@ -36,17 +33,17 @@ class Overview extends React.Component {
     return (
       <div className="Overview" data-testid="Overview">
         <div>Overview Here!</div>
+        {this.props.renderStars()}
         <div className="ProductTitle" >{this.state.info.name}</div>
         <div className="Category">{this.state.info.category}</div>
         {/* <div>OV Price: {this.state.info.default_price}</div> */}
```

Comment:
Can we get rid of this comment?

Reply:
I need this as a placeholder reminder. I got fulfilled all the other comments though.